dressup/src/pages/Homepage/Footer/Footer.js

```diff
@@ -0,0 +1,32 @@
+import "./Footer.css";
+
+function Footer() {
+  return (
+    <div className="footer">
+      <div class="shop-by">
+        <p class="shop-by-heading">SHOP BY</p>
+        <p>Men</p>
+        <p>Women</p>
+        <p>Kids</p>
+        <p>New Arrivals</p>
+      </div>
+
+      <div class="dress-up">
+        <p class="dress-up-heading">DRESS UP</p>
+        <p>Contact us</p>
+        <p>FAQ</p>
+        <p>Track Orders</p>
+        <p>Cancellation</p>
+      </div>
+
+      <div class="about-us">
+        <p class="about-us-heading">ABOUT US</p>
+        <p>blogs</p>
+        <p>Contact</p>
+        <p>Careers</p>
+      </div>
```

Comment:
you can turn it into a page specific component, there is a clear pattern of 1 <div> and 5 <p> tags